File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/ExceptionHandler.vb
Imports Autodesk.AutoCAD.ApplicationServices
Imports Autodesk.AutoCAD.EditorInput

Public Class ExceptionHandler
    Public Shared Sub Handle(ex As Exception)
        Dim doc As Document = Core.Application.DocumentManager.MdiActiveDocument
        Dim ed As Editor = doc.Editor
        ed.WriteMessage(Environment.NewLine.ToString() & "Exception: " + ex.Message)
    End Sub

End Class

File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/ExceptionHandler.cs
using System;

namespace cadwiki.AutoCAD2021.Base.Utilities
{

    public class ExceptionHandler
    {
        public static void WriteToEditor(Exception ex)
        {
            var doc = global::Autodesk.AutoCAD.ApplicationServices.Core.Application.DocumentManager.MdiActiveDocument;
            var ed = doc.Editor;
            var list = NetUtils.Exceptions.GetPrettyStringList(ex);
            foreach (string str in list)
                ed.WriteMessage(Environment.NewLine.ToString() + str);
        }



    }
}